_pages/fsa/features/dynamicforms/core/form-controls-document-upload.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Document Upload
+---
+
+**Note**: This feature is introduced with version 2.0 of the FSA Spartacus libraries.
+
+## Contents
+
+- [Overview](#overview)
+- [Document Upload Usage](#document-upload-usage)
+- [File Upload](#file-upload) 
+
+## Overview
+
+The document upload feature is one of the many components implemented in the Dynamicforms library and is used to make a form control that stores an image-kind of data. The cx-upload is used for uploading one or multiple images, documents, audio, video, and other files to a server. It provides a rich set of features like multiple file selection, real-time progress bar, document download, document deletion, file validation, and more. This cx-upload component uses the Document Upload API for uploading files.
```

Comment:
> used to make a form control that stores an image-kind of data. The cx-upload is
I would remove this chunk entirely, since it's not completely correct, and is in conflict with the next sentence.
Perhaps have it like this:
```The document upload feature is one of the many components implemented in the Dynamicforms library and  is used to enable the user to upload one or multiple images, documents, audio, video, and other file types to a server.```

Reply:
Agree, content is fixed now.